lux/core/frame.py

```diff
@@ -609,7 +611,9 @@ def on_button_clicked(b):
 					
 		except(KeyboardInterrupt,SystemExit):
 			raise
-		except:
+		except Exception as e:
+			warnings.warn(e)
+			warnings.warn(traceback.format_exc())
```

Comment:
What does the `format_exc` do here?

Reply:
It just gives the full stack trace rather than just saying that lux-widget failed. This is just more robust issue reporting so if users encounter this, they know what to post on a Github issue. It just makes this show up as a more normal python error while also letting the user continue using the pandas view. I doubt that this will be used often if we put out compatible lux and lux-widget versions and stay up to date on requirements.txt, but in the case where it does fail, this will give much more insight into why.

Reply:
Ok that makes sense. I've modified this a bit so that the `warnings.warn(e)` doesn't throw an error.